lib/src/logger/stderr.dart

```diff
@@ -9,8 +9,9 @@ import 'package:stack_trace/stack_trace.dart';
 import '../io.dart';
 import '../logger.dart';
 import '../utils.dart';
+import '../node/console_stub.dart' if (dart.library.js) '../node/console.dart';
```

Comment:
@nex3 Is there a better way to do this that still avoids `Dart library 'dart:js_util' is not available on this platform` errors?

Reply:
Unfortunately no :(. The Dart VM still needs to know how to typecheck the APIs even if it won't actually run them in practice.

I think, on retrospect, that the cleanest approach here is probably to hide even more of the implementation and just expose a `printError()` function. I don't want to make you undo all the refactors so I've just gone and made the change myself.

Reply:
@nex3 Makes sense, and LGTM! 🚀